cmd/analyzeRepoStaleBranches.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var threadsRepoStaleBranch int
+var expand bool
+var regex string
+
+// analyzeRepoCmd represents the analyzeRepo command
```

Comment:
```suggestion
```
nitpick : I think this comment should be removed not relevant for this , probably copy paste ?